frontend/src/reducer/reducer.js

```diff
@@ -36,6 +36,14 @@ const reducer = (state = initialState, action) => {
         ...newState
       };
 
+    case "FETCHING_USER_DATA":
```

Comment:
Whet does this do ? I'm honestly confused

Reply:
that replace the last value by the correct user value